### ISSUE TITLE
Fatal error: boost::bad_any_cast: failed conversion using boost

### ISSUE THREAD

**Issue by abhis27** [forknote/forknote#15]
After successful compile of build, i am having cast exception in following command:

$ `./walletd --container-file=test123 --container-password=test123 --generate-container`

> `Fatal error: boost::bad_any_cast: failed conversion using boost::any_cast`

_( I have tried both v2.0.1 & v2.0.4 & i am doing with libboost-all-dev==1.58.0 + Ubuntu==16.04)_

**Comment by pmitchev** [comment]
Indeed, I got the same error. It will be fixed ASAP. In the meantime you can use walletd from v2.0.1.1

**Comment by pmitchev** [comment]
I just realized you are not using configuration file. Configuration files are mandatory with all binaries.

Also, you must add container-file and container-password into the configuration file:
```
container-file=/full/path/test.wallet
container-password=test123

```

Then you use like this:
`./walletd -c config.file -g
`

Anyways, the error you are getting is not informative at all and it will be fixed in the next version

**Comment by pmitchev** [comment]
Here is a full config file example for the Dashcoin network:

`BYTECOIN_NETWORK=12112111-1110-4101-1311-001212110110
CRYPTONOTE_DISPLAY_DECIMAL_POINT=12
CRYPTONOTE_NAME=dashcoin
CRYPTONOTE_PUBLIC_ADDRESS_BASE58_PREFIX=72
GENESIS_COINBASE_TX_HEX=010a01ff0001ffffffffffff0f029b2e4c0271c0b42e7c53291a94d1c0cbff8883f8024f5142ee494ffbbd08807121013c086a48c15fb637a96991bc6d53caf77068b5ba6eeb3c82357228c49790584a
MAX_BLOCK_SIZE_INITIAL=25600
P2P_STAT_TRUSTED_PUB_KEY=

DEFAULT_FEE=1000000000
MAX_TRANSACTION_SIZE_LIMIT=19400

UPGRADE_HEIGHT_V2=91452
UPGRADE_HEIGHT_V3=4294967295

p2p-bind-port=29080
rpc-bind-port=29081

CHECKPOINT=28000:70d2531151529ac00bf875281e15f51324934bc85e5733dcd92e1ccb1a665ff8
CHECKPOINT=40000:c181ec9223a91fef8658c7aa364c093c41c28d250870ca1ed829bf74f0abf038
CHECKPOINT=55000:5289fe9f2dce8f51441019b9fbc85c70ad85ff49a666ef0109f3269890c6af6d
CHECKPOINT=70000:193e335f34b8b8f1fab3857111cb668c2720340e80176a25155071e573481acb
CHECKPOINT=87500:cce8a035f34457ec1098ab41e5949cac3db00ebff3503e26f36bfa057543095a
CHECKPOINT=150000:118989862ef4903df981501b8e01fe76ee15ab960d52
CHECKPOINT=200000:bb5b339991347c3b06431b233d3dbf9201e9176ddf4349ee46c60473725d5c98
CHECKPOINT=250000:c5b6064258237e911300cdcb6f937b37a6c17932e2bfdcba0ff12a48694709e0
CHECKPOINT=298000:5b1f21234c911eb755abce6fa66358558b380526f606222d042eb8b7ce993ae8
CHECKPOINT=360000:d9426fcdaa95a8dd2f1d151c2088a0dac2143963b8088d0de02f9c08ec8429f5
CHECKPOINT=423000:7e2c45f1a546290a7f7f28cb83c1bdc21f531f410ad0e65a12a5c86b95e194b5
CHECKPOINT=529000:b0bbedbd3e8d8051b273a8a7c8e875e1ad4184c37e6cedfb68d930e8235153dd
CHECKPOINT=600000:1f265d7710dc8b28478c62d48ef533e462008068cd7a5912b861c350ce82fbae
CHECKPOINT=636000:436f4ed7684698160120f25f1bad8aa92c38c93fbedd2b0c74671adf4d4fb191
CHECKPOINT=656000:cebe59fafe61f901dea398c0de3e7979171b67fabc31270514e261f1ae633d9f

seed-node=108.61.188.93:7610
seed-node=128.199.146.243:29080
seed-node=195.154.181.121:7280
seed-node=93.77.229.33:29080
seed-node=144.76.59.211:29080
seed-node=176.9.147.178:7280
seed-node=176.9.47.243:7280
seed-node=84.200.76.33:29080
seed-node=98.115.76.178:29080
seed-node=195.154.181.121:7280

container-file = /root/forknote/dashcoin.mycontainer 
container-password = test123
daemon-port = 29081
bind-port = 9090
SYNC_FROM_ZERO = 1`

**Comment by abhis27** [comment]
Fixed, in 2.0.6 (y)

**Comment by Xuiquaxa** [comment]
Failed to parse arguments: unrecognised option 'BYTECOIN_NETWORK'
it's show while strar my daemon, do you know my issue?
